### PR TITLE
add blogpost to jdbc to bq template

### LINKED ISSUE #152: [publish] Blog on JDBCToBigQuery

### LINKED ISSUE #304: [publish] Blog on JDBCToBigQuery

### PR DESCRIPTION
**Pull request by sjlva**
Closes #152 and closes #304

### CODE REVIEW

**Comment by shashank-google** [comment]
Thanks @sjlva for the contribution.
@hhasija Is it possible for you to review the blog posts and share your comments if any here.

@sjlva Once reviewed we want this blog post to [GCP Community](https://medium.com/google-cloud).
And publish medium post URL with GCP Community.

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Please refer to the [Dataproc Templates (Java - Spark) README](/java/README.md)
 * [HBaseToGCS](/java/src/main/java/com/google/cloud/dataproc/templates/hbase/README.md)
 * [SpannerToGCS](/java/src/main/java/com/google/cloud/dataproc/templates/databases/README.md) (blogpost [link](https://medium.com/google-cloud/cloud-spanner-export-query-results-using-dataproc-serverless-6f2f65b583a4))
 * [S3ToBigQuery](/java/src/main/java/com/google/cloud/dataproc/templates/s3/README.md)
-* [JDBCToBigQuery](/java/src/main/java/com/google/cloud/dataproc/templates/jdbc/README.md)
+* [JDBCToBigQuery](/java/src/main/java/com/google/cloud/dataproc/templates/jdbc/README.md) (blogpost [link](https://medium.com/@sjlva/java-fast-export-large-database-tables-using-gcp-serverless-dataproc-fe6ffffe28b5))
```
Comment (shashank-google) [inline README.md:25]: same changes need to go into /java/README.MD

File: README.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ Please refer to the [Dataproc Templates (Python - PySpark) README](/python/READM
 * [SnowflakeToGCS](/python/dataproc_templates/snowflake/README.md)
 * [JDBCToJDBC](/python/dataproc_templates/jdbc/README.md)
 * [JDBCToGCS](/python/dataproc_templates/jdbc/README.md)
-* [JDBCToBigQuery](/python/dataproc_templates/jdbc/README.md)
+* [JDBCToBigQuery](/python/dataproc_templates/jdbc/README.md) (blogpost [link](https://medium.com/@sjlva/python-fast-export-large-database-tables-using-gcp-serverless-dataproc-bfe77a132485))
```
Comment (shashank-google) [inline README.md:51]: same changes need to go into /python/README.MD

**Review by shashank-google — CHANGES_REQUESTED** [review]

**Review by shashank-google — APPROVED** [review]

**Comment by shashank-google** [comment]
@sjlva 
I just realized your medium posts links are still not show GCP Community.
Have you submitted it ? If not please do so.

**Comment by hhasija** [comment]
> @sjlva I just realized your medium posts links are still not show GCP Community. Have you submitted it ? If not please do so.



> @sjlva I just realized your medium posts links are still not show GCP Community. Have you submitted it ? If not please do so.

@sjlva Please let me know, in case if you need any help in publishing those blog posts to GCP Community.

**Comment by sjlva** [comment]
> > @sjlva I just realized your medium posts links are still not show GCP Community. Have you submitted it ? If not please do so.
> 
> > @sjlva I just realized your medium posts links are still not show GCP Community. Have you submitted it ? If not please do so.
> 
> @sjlva Please let me know, in case if you need any help in publishing those blog posts to GCP Community.

@hhasija I asked alexisimp on Twitter. This is the only way to publish those to GCP Community, right?

**Comment by sjlva** [comment]
> > @sjlva I just realized your medium posts links are still not show GCP Community. Have you submitted it ? If not please do so.
> 
> > @sjlva I just realized your medium posts links are still not show GCP Community. Have you submitted it ? If not please do so.
> 
> @sjlva Please let me know, in case if you need any help in publishing those blog posts to GCP Community.

@hhasija Done. Posts in GCP Community :)

**Comment by hhasija** [comment]
> > > @sjlva I just realized your medium posts links are still not show GCP Community. Have you submitted it ? If not please do so.
> > 
> > 
> > > @sjlva I just realized your medium posts links are still not show GCP Community. Have you submitted it ? If not please do so.
> > 
> > 
> > @sjlva Please let me know, in case if you need any help in publishing those blog posts to GCP Community.
> 
> @hhasija Done. Posts in GCP Community :)

Looks good @sjlva . If you are done with blog posts and also if you have updated the Read me files as well. Then I would suggest you to merge this pull request, so that corresponding issues would get closed automatically. 

Really thankful for your contributions.

**Comment by sjlva** [comment]
> > > > @sjlva I just realized your medium posts links are still not show GCP Community. Have you submitted it ? If not please do so.
> > > 
> > > 
> > > > @sjlva I just realized your medium posts links are still not show GCP Community. Have you submitted it ? If not please do so.
> > > 
> > > 
> > > @sjlva Please let me know, in case if you need any help in publishing those blog posts to GCP Community.
> > 
> > 
> > @hhasija Done. Posts in GCP Community :)
> 
> Looks good @sjlva . If you are done with blog posts and also if you have updated the Read me files as well. Then I would suggest you to merge this pull request, so that corresponding issues would get closed automatically.
> 
> Really thankful for your contributions.

@hhasija I do not have permissions to merge.

**Pull request merged**